app/lib/bk/compat/parsers/circleci/orbs.rb

```diff
@@ -28,6 +19,55 @@ def translator(action, _config)
           ]
         end
       end
+
+      # Docker orb specific implementation
+      class DockerOrb
+        def initialize
+          @config = {}
+        end
+
+        def matcher(orb, _conf)
+          orb.start_with?('docker/')
+        end
+
+        def translator(action, config)
+          @config = config
+          method_map = {
+            'docker/build' => -> { translate_docker_build(config) },
+            'docker/install-docker' => -> { validate_and_translate_install_docker(config) }
+          }
+          method_map.fetch(action, -> { ["# Unsupported docker orb action: #{action}"] }).call
+        end
+
+        private
+
+        def validate_and_translate_install_docker(_config)
+          install_dir = @config['install-dir'] || '/usr/local/bin'
+          version = @config['version'] || 'latest'
```

Comment:
just for consistency with the rest of the codebase I would recommend changing these constructs to use `fetch` instead (take into account a previous comment about not using the instance variable as well):

```suggestion
          install_dir = @config.fetch('install-dir', '/usr/local/bin')
          version = @config.fetch('version', 'latest')
```